src/bpf/bpfd-manager.c

```diff
@@ -0,0 +1,448 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/wait.h>
+
+#include "bpf-dlopen.h"
+#include "fileio.h"
+#include "sd-daemon.h"
+#include "sd-messages.h"
+#include "bpfd-manager.h"
+#include "fd-util.h"
+#include "fs-util.h"
+#include "hashmap.h"
+#include "missing_syscall.h"
+#include "mkdir.h"
+#include "process-util.h"
+#include "signal-util.h"
+#include "socket-util.h"
+#include "stdio-util.h"
+#include "umask-util.h"
+
+static int get_memlock_by_fd(int fd, char **memlock) {
```

Comment:
we have this coding style rule that return parameters should be called `ret` or `ret_xyz`. the 2nd parameter here is a return param, hence please rename to `ret` or to `ret_memlock` please